src/main/java/net/dv8tion/jda/client/entities/Group.java

```diff
@@ -39,9 +40,9 @@
     List<User> getUsers();
     List<User> getNonFriendUsers();
     List<Friend> getFriends();
-
-    //getGroupManager()
-
+    
+    GroupManager getGroupManager();
```

Comment:
If a class has a Manager, it has an updatable one. No clue what you were looking at.